src/runtime/texture.h

```diff
@@ -94,6 +94,25 @@ inline bool IsTextureStorage(std::string scope) {
   return scope.find("texture") != std::string::npos;
 }
 
+class TVM_DLL Pool {
```

Comment:
Pool -> TexturePool? since this class is responsible only for caching of textures, not buffers
Since we are moving to header :)

Reply:
This class is used in the class `TexturePool`. Class `TexturePool` is defined below (line 121). Probably we could rename both of these classes.

Reply:
this class is declared under tvm::runtime namespace that means that is visible and can be acceptable for anyone + gives impression that there is something generic in context of the whole runtime while it is something special for opencl textures only

Reply:
Renamed to `Pool2D`